### PR TITLE
feat: solution page block updates exlm-889

### PR DESCRIPTION
**Pull request by nitin-rachabathuni**
Please always provide the Jira Issue your PR is for, as well as test URLs where your change can be observed (before and after):

Jira ID: https://jira.corp.adobe.com/browse/EXLM-889

Test URLs:

- Before: https://main-ui--exlm--adobe-experience-league.hlx.live/en/docs/home-tutorials
- After: https://feature-exlm-889-block-updates--exlm--adobe-experience-league.hlx.live/en/docs/home-tutorials

### CODE REVIEW

File: blocks/list/list.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
+.docs .list-header-box {
```
Comment (ruhisingh1) [inline blocks/list/list.css:1]: Please target "docs-solution-landing" class instead of "docs"

File: blocks/list/list.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import { isDocPage } from '../../scripts/scripts.js';
+
+export default async function decorate(block) {
+  if (!isDocPage()) {
```
Comment (ruhisingh1) [inline blocks/list/list.js:4]: Please check only for "docs-solution-landing" page. Exclude docs pages.

File: blocks/list/list.css (hunk 1 above)
Comment (nitin-rachabathuni) [reply blocks/list/list.css]: Thanks Ruhi, done

File: blocks/list/list.js (hunk 2 above)
Comment (nitin-rachabathuni) [reply blocks/list/list.js]: Thanks Ruhi, done

**Review by Sivaramvt — APPROVED** [review]

**Pull request merged**